.teamcity/src/subprojects/build/core/NativeBuild.kt

```diff
@@ -0,0 +1,25 @@
+package subprojects.build.core
+
+import jetbrains.buildServer.configs.kotlin.v10.*
+import jetbrains.buildServer.configs.kotlin.v2019_2.BuildType
+import jetbrains.buildServer.configs.kotlin.v2019_2.buildSteps.*
+import subprojects.*
+
+class NativeBuild(private val osEntry: OSEntry) : BuildType({
+    id("KtorMatrixNative_${osEntry.name}".toExtId())
+    name = "Native on ${osEntry.name}"
+
+    setupDefaultVCSRootAndTriggers()
+
+    steps {
+        gradle {
+            name = "Build and Run Tests"
+            tasks = "ktor-client:ktor-client-curl:${osEntry.taskName} --info"
```

Comment:
Is it enough to just build `ktor-client-curl` for the native?

Reply:
I believe so, yes